apps/www/components/Pricing/PricingComparisonTable.tsx

```diff
@@ -327,54 +362,70 @@ const PricingComparisonTable = () => {
-                        {['Free', 'Pro', 'Team'].includes(plan.name) && (
-                          <span className="text-[13px] leading-4 mt-1">{plan.costUnit}</span>
-                        )}
-                      </p>
-                    </span>
-                    <span className="flex flex-col justify-between h-full pb-2">
-                      <Button
-                        asChild
-                        size="tiny"
-                        type={plan.name === 'Enterprise' ? 'default' : 'primary'}
-                        block
-                      >
-                        <Link
-                          href={plan.href}
-                          onClick={() =>
-                            sendTelemetryEvent(
-                              gaEvents[`www_pricing_comparison_${plan.name.toLowerCase()}`]
-                            )
-                          }
+              {plans.map((plan) => {
+                const isUpgradablePlan = plan.name === 'Pro' || plan.name === 'Team'
+
+                return (
+                  <th
+                    className="text-foreground w-1/4 px-0 text-left text-sm font-normal"
+                    scope="col"
+                    key={plan.name}
+                  >
+                    <span className="flex flex-col px-6 pr-2 pt-2 gap-1.5">
+                      <span className="flex flex-col xl:flex-row xl:items-end gap-1">
+                        <h3 className="text-lg xl:text-xl 2xl:text-2xl leading-5 uppercase font-mono font-normal flex items-center">
+                          {plan.name}
+                        </h3>
+                        <p
+                          className={cn(
+                            'text-foreground-lighter -my-1 xl:m-0',
+                            plan.name === 'Enterprise' && 'xl:opacity-0'
+                          )}
                         >
-                          {plan.cta}
-                        </Link>
-                      </Button>
+                          <span className="text-foreground-lighter font-mono text-xl mr-1 tracking-tighter">
+                            {plan.name !== 'Enterprise' && '$'}
+                            {plan.priceMonthly}
+                          </span>
+                          {['Free', 'Pro', 'Team'].includes(plan.name) && (
+                            <span className="text-[13px] leading-4 mt-1">{plan.costUnit}</span>
+                          )}
+                        </p>
+                      </span>
+                      <span className="flex flex-col justify-between h-full pb-2">
+                        {isUpgradablePlan && hasExistingOrganizations ? (
+                          <UpgradePlan
+                            organizations={organizations}
+                            onClick={() =>
+                              sendTelemetryEvent(
+                                gaEvents[`www_pricing_comparison_${plan.name.toLowerCase()}_mobile`]
```

Comment:
This should be "not" `_mobile`, like line 416.
The "Upgrade plan"s could have their own separate event btw.

Reply:
Nice catch! Removed `_mobile` there and added a new event for the upgrade buttons :)